src/workerd/tests/test-fixture.c++

```diff
@@ -230,11 +231,73 @@ struct MockResponse final: public kj::HttpService::Response {
     KJ_FAIL_REQUIRE("NOT SUPPORTED");
   }
 };
+
+class MockLoopback : public Worker::Actor::Loopback, public kj::Refcounted {
+  public:
+  virtual kj::Own<WorkerInterface> getWorker(IoChannelFactory::SubrequestMetadata metadata) {
+    return kj::Own<WorkerInterface>();
+  };
+
+  virtual kj::Own<Worker::Actor::Loopback> addRef() {
+    return kj::addRef(*this);
+  };
+};
+
+// An ActorStorage implementation which will always respond to reads as if the state is empty,
+// and will fail any writes.
+//
+// TODO(cleanup) This is an exact copy of the implementation in src/workerd/server/server.c++.
+// Should that one be exported and used here too?
```

Comment:
Re: the question:
> This is an exact copy of the implementation in src/workerd/server/server.c++.
> Should that one be exported and used here too?

I'd generally prefer that to having a duplicate.

Reply:
Done.